handel/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./bin/handel",
   "scripts": {
     "lint": "tslint -p tsconfig.json -t stylish",
-    "build": "tsc",
+    "build": "gulp build",
```

Comment:
I had accidentally changed this in the last PR, which broke some stuff.  Ooops 😖